scripts/configure_invokeai.py

```diff
@@ -706,7 +706,8 @@ def main():
                         type=str,
                         default='./configs/models.yaml',
                         help='path to configuration file to create')
-    parser.add_argument('--root',
+    parser.add_argument('--root_dir',
+                        '--root',
```

Comment:
Thanks for catching this. We use `--root_dir` in a few places, so this does indeed make it more consistent.

```suggestion
    parser.add_argument('--root_dir',
```

Suggestion: you can just replace `--root` with `--root_dir`. The `--root` argument will continue to work, because `argparse` will use the shortest unique matching substring. If you're okay with this, let's apply it.

Thanks for the contribution!